test/include.test.js

```diff
@@ -1254,6 +1254,43 @@ describe('include', function() {
     });
   });
 
+  it('should not throw on fetch User if include is boolean equals true', function(done) {
+    User.find({include: true}, function(err, users) {
+      should.not.exist(err);
```

Comment:
```suggestion
      if (err) return done(err);
```

This way the original stack trace is preserved to make troubleshooting easier.